src/partials/footer.html.eco

```diff
@@ -23,6 +23,7 @@
 <script src="<%= @site.url %>/vendor/respond/respond.src.js"></script>
 <![endif]-->
 
+<script src="<%= @site.url %>/js/events.js"></script>
```

Comment:
Should this be loaded in every page? Please move to `src/documents/events/index.html.eco`.